packages/mui-material-next/src/FormControl/FormControl.test.tsx

```diff
@@ -47,15 +58,15 @@ describe('<FormControl />', () => {
       const root = container.firstChild;
 
       expect(root).not.to.have.class(classes.marginNormal);
-      expect(root).not.to.have.class(classes.sizeSmall);
+      expect(root).not.to.have.class('MuiFormControl-sizeSmall');
```

Comment:
Was this class removed?

Reply:
It was [never there](https://github.com/mui/material-ui/blob/master/packages/mui-material/src/FormControl/formControlClasses.ts) to begin with! So I hardcoded the string as TS was complaining it doesn't exist

Reply:
Should we remove the `expect(...`? 😅

Reply:
Actually looking back at the [PR](https://github.com/mui/material-ui/pull/23832) when the `size` classes were introduced, I don't see why `FormControl` shouldn't have the `sizeSmall` class, and we should add it back ~

It's different from the `margin` prop (that it replaced in the test), and it's companion components (inputs, labels etc) apply it when `size="small"`

Reply:
I've added a task in https://github.com/mui/material-ui/issues/38411 to do in a separate PR